packages/code/src/CopyButton.tsx

```diff
@@ -2,69 +2,64 @@ import React, { useState, useEffect } from 'react';
 import ClipboardJS from 'clipboard';
 import { VisuallyHidden } from '@leafygreen-ui/a11y';
 import { cx, css } from '@leafygreen-ui/emotion';
-import { palette, uiColors } from '@leafygreen-ui/palette';
+import { palette } from '@leafygreen-ui/palette';
 import CheckmarkIcon from '@leafygreen-ui/icon/dist/Checkmark';
 import CopyIcon from '@leafygreen-ui/icon/dist/Copy';
 import IconButton from '@leafygreen-ui/icon-button';
-import { Mode } from './types';
 import { usePopoverPortalContainer } from '@leafygreen-ui/leafygreen-provider';
+import { Theme } from '@leafygreen-ui/lib';
+import { useCodeContext } from './CodeContext';
 
-const copiedStyle = css`
-  color: ${palette.white};
-  background-color: ${palette.green.dark1};
-
-  &:focus {
+const copiedThemeStyle: Record<Theme, string> = {
+  [Theme.Light]: css`
     color: ${palette.white};
+    background-color: ${palette.green.dark1};
 
-    &:before {
-      background-color: ${palette.green.dark1};
-    }
-  }
-
-  &:hover {
-    color: ${palette.white};
+    &:focus,
+    &:hover {
+      color: ${palette.white};
 
-    &:before {
-      background-color: ${palette.green.dark1};
+      &:before {
+        background-color: ${palette.green.dark1};
+      }
```

Comment:
From Figma, I think this is supposed to be `green.base` 
<img width="111" alt="Screen Shot 2022-11-02 at 4 41 36 PM" src="https://user-images.githubusercontent.com/2414030/199598500-ed8fccdc-f3ce-4371-a639-8475f91c6d71.png">
![Screen Shot 2022-11-02 at 4 41 32 PM](https://user-images.githubusercontent.com/2414030/199598501-994ded6d-3c3e-4fe7-a55e-8e44cb47dcc3.png)